AGM_Interaction/config.cpp

```diff
@@ -334,7 +359,7 @@ class CfgVehicles {
     class AGM_SelfActions {
       class AGM_TeamManagement {
         displayName = "$STR_AGM_Interaction_TeamManagement";
-        condition = "missionNamespace getVariable ['AGM_enableTeamManagement', true]";
+        condition = "missionNamespace getVariable ['AGM_enableTeamManagement', 1] == 1";
```

Comment:
BTW, this:

`missionNamespace getVariable ['AGM_enableTeamManagement', 1] == 1`

is not really needed. Can be replaced by:

`AGM_enableTeamManagement > 0`

because parameters are guaranteed to be always defined after server init.